pymc3/tests/test_model.py

```diff
@@ -179,3 +182,80 @@ def test_empty_observed():
         npt.assert_allclose(a.tag.test_value, np.zeros((2, 3)))
         b = pm.Beta('b', alpha=1, beta=1, observed=data)
         npt.assert_allclose(b.tag.test_value, np.ones((2, 3)) / 2)
+
+
+class TestValueGradFunction(unittest.TestCase):
```

Comment:
Ah, I didn't really think about this but just used the `unittest` version. pytest supports this though. I guess the alternative would be to use a pytest fixture? So you'd use that?